app/assets/stylesheets/petitions/_forms.scss

```diff
@@ -38,16 +43,13 @@ select.form-control {
 
 .character-count {
   color: $secondary-text-colour;
-  text-align: right;
+  float: right;
```

Comment:
Yeah I reckon so. If mobile devices have average js support, around 1% of users will see that. It's functional.